### ISSUE TITLE
Question about the baseline_aic_finetune.yml config file

### ISSUE THREAD

**Issue by clw5180** [heshuting555/AICITY2020_DMT_VehicleReID#1]
Very nice project !  I compare with https://github.com/michuanhaohao/reid-strong-baseline/blob/master/configs/softmax_triplet.yml , and see some differences such as 

IF_LABELSMOOTH: 'off'      # origin is 'on'
SIZE_TRAIN: [320, 320]       # origin is [256, 128]
 OPTIMIZER_NAME: 'SGD'  # origin is 'Adam'
  MAX_EPOCHS: 40             # origin is 120
  BASE_LR: 0.001                 # origin is 0.00035
  IMS_PER_BATCH: 96          # origin is 64
......

Can you share some experiences about how to choose these hyp? Thanks a lot !!

**Comment by clw5180** [comment]
And do you training on the VeRi dataset from scratch first as pretrained model, and then fine-tune on the competition dataset?  Thanks

**Comment by heshuting555** [comment]
Thank you for your question, the competition requires that no external data can be used, but the virtual data provided by the competition can be used. So we first train a model based on virtual data and competition data, and then finetune the model on the competition data.

**Comment by heshuting555** [comment]
Regarding the setting of parameters, this is because the task of vehicle reid and the task of person reid are similar, but there are still differences. The parameters should be selected according to the actual data set. For the finetune process, we set the learning rate to one-tenth of the normal training process to ensure it converges better.
